File: src/main/java/net/auoeke/limitless/config/enchantment/entry/normalization/EnchantmentNormalizationEntry.java
package net.auoeke.limitless.config.enchantment.entry.normalization;

import me.shedaniel.autoconfig.annotation.ConfigEntry.BoundedDiscrete;

public class EnchantmentNormalizationEntry {
    public boolean enabled = true;

    @BoundedDiscrete(max = 200)
    public int offset = 30;
}

File: src/net/auoeke/limitless/config/enchantment/entry/normalization/EnchantmentNormalizationEntry.kt
package net.auoeke.limitless.config.enchantment.entry.normalization

import me.shedaniel.autoconfig.annotation.ConfigEntry.BoundedDiscrete

class EnchantmentNormalizationEntry {
    var enabled = true

    @BoundedDiscrete(max = 200)
    var offset = 30
}
